java/odsa-spring/src/main/java/com/oracle/dev/jdbc/odsa/spring/Employee.java

```diff
@@ -0,0 +1,115 @@
+/*
+  Copyright (c) 2021, 2022, Oracle and/or its affiliates.
+
+  This software is dual-licensed to you under the Universal Permissive License
+  (UPL) 1.0 as shown at https://oss.oracle.com/licenses/upl or Apache License
+  2.0 as shown at http://www.apache.org/licenses/LICENSE-2.0. You may choose
+  either license.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+     https://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+
+package com.oracle.dev.jdbc.odsa.spring;
+
+import java.util.Objects;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+
+@Entity
+@Table(name = "EMPLOYEE", schema = "ADMIN")
```

Comment:
The ADMIN schema should not be used for user/developer resources. As our code samples show the best practices, I'd suggest using a different schema. There use to be an HR/HR schema for demo purposes but I am not sure it's still around on Autonomous DB. If not you can ask the user to create one using SQL Developer Web interface (available on ADB console).

Reply:
fixed